pkg/mountutil/volumestore/volumestore.go

```diff
@@ -22,20 +22,38 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"regexp"
+	"strings"
 
 	"github.com/containerd/containerd/identifiers"
 	"github.com/containerd/errdefs"
 	"github.com/containerd/log"
 	"github.com/containerd/nerdctl/v2/pkg/inspecttypes/native"
 	"github.com/containerd/nerdctl/v2/pkg/lockutil"
 	"github.com/containerd/nerdctl/v2/pkg/strutil"
+	"github.com/opencontainers/go-digest"
 )
 
 const (
 	dataDirName        = "_data"
 	volumeJSONFileName = "volume.json"
 )
 
+var (
+	validateRegExp = regexp.MustCompile("^[a-zA-Z0-9_+:/-]{1,256}$")
+)
+
+func validateName(name string) error {
```

Comment:
Purely provisional ^ - to be replaced by whatever we deem right.
I do not quite care what this does exactly, as this PR technically should free us from any limitations on names.